.gitignore

```diff
@@ -6,3 +6,4 @@ server/config.txt
 player_scores.json
 
 server/ai_agent.py
+agents/ai_agent.py
```

Comment:
Let's move agents to either client/agents or common/agents if it's needed by the server.